### PR TITLE
NEXT-14511 - API versioning

### PR DESCRIPTION
**Pull request by Dominik28111**

### CODE REVIEW

File: guides/integrations-api/general-concepts/README.md [hunk 1]
```diff
@@ -20,5 +20,8 @@ And a common approach regarding the way that Shopware provides endpoint referenc
 
 {% page-ref page="generated-reference.md" %}
 
+## API Versioning
 
 
+
+{% page-ref page="api-versioning.md" %}
```
Comment (Dominik28111) [inline guides/integrations-api/general-concepts/README.md:27]: Any idea what we can write here? 😄

Comment (Dominik28111) [inline guides/integrations-api/general-concepts/README.md:27]: Any idea what I can write here? 😄

File: guides/integrations-api/general-concepts/api-versioning.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+You can read the currently used version in the API as well. Starting with Shopware 6.3.5.0, you can use this route to fetch the current version:
+`GET /api/_info/version`
+
+Prior to that, the version was readable using the following route:
+`GET /api/v2/_info/config`
+
+### How do I actually open up the Swagger page?
+
+Simply navigate to the following URL in your shop: `//yourShop.com/api/_info/swagger.html`
```
Comment (lernhart) [inline guides/integrations-api/general-concepts/api-versioning.md:73]: You did relative paths in all of the article. Why use a somewhat absolute path here ?

```suggestion
Simply navigate to the following URL in your shop: `/api/_info/swagger.html`
```

**Review by lernhart — CHANGES_REQUESTED** [review]

**Comment by JoshuaBehrens** [comment]
I am missing the new changes from @shyim (and sort-of me) how to still validate a shop version in a single request.

**Comment by PaddyS** [comment]
@JoshuaBehrens They're still in internal code review, but they'll be here as well. :-)

**Comment by PaddyS** [comment]
This should be part of the 6.4.0.0 branch though. We'll leave this one open for now

File: guides/integrations-api/general-concepts/README.md (hunk 1 above)
Comment (PaddyS) [reply guides/integrations-api/general-concepts/README.md:27]: `Starting with Shopware version 6.4.0.0, we decided to change our API versioning strategy. 
The following article will cover what has been done and changed, how it used to be and how the version strategy looks like now.`


Copied it from the article itself and modified a bit.

**Comment by leichteckig** [comment]
`master` is version 6.4.0.0 now.  Please retarget this MR to `master` and rebase, if needed.

File: guides/integrations-api/general-concepts/api-versioning.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# New API versioning explained
+
+## Overview
+
+Starting with Shopware version 6.4.0.0, we decided to change our API versioning strategy. 
+This article will cover what has been done and changed, how it used to be and how the version strategy looks like now.
+
+### Versioning prior to 6.3.5.0
```
Comment (lernhart) [inline guides/integrations-api/general-concepts/api-versioning.md:8]: Starting with Shopware 6.4.0.0, we decided [...]

Versioning prior to 6.3.5.0 

So what now ? Did we change this in 6.4.0.0 or 6.3.5.0 ?

**Review by lernhart — CHANGES_REQUESTED** [review]
Please rebase to master

**Review by PaddyS — APPROVED** [review]

**Review by lernhart — APPROVED** [review]

**Pull request merged**